Dockerfile.assisted-service

```diff
@@ -45,15 +45,11 @@ RUN dnf install -y libvirt-libs python3 && \
     dnf install -y NetworkManager-config-server && \
     dnf install -y openvswitch2.11 && \
     dnf install -y python3-openvswitch2.11 && \
+    dnf install -y nmstate \
```

Comment:
@yevgeny-shnaidman I think there was a specific reason you used the python lib right?
Maybe due to the RPM being old version

anyway, I think we must pin the version here, no?

Reply:
Yes, the reason the dockerfile is using the python package is because the RPM version is old. We must make sure the RPM is updated so we can switch to using it. 

I will pin the version as soon as there's one available. 

As it is right now, this PR is just a placeholder to make sure we don't forget :smiley:  (in addition to the issue created by @nmagnezi )